python/lsst/analysis/tools/actions/vector/calcBinnedStats.py

```diff
@@ -0,0 +1,97 @@
+# This file is part of analysis_tools.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+from __future__ import annotations
+
+__all__ = ("CalcBinnedStatsAction",)
+
+from functools import cached_property
+from typing import cast
+
+import numpy as np
+from lsst.pex.config import Field
+from lsst.pipe.tasks.configurableActions import ConfigurableActionField
+
+from ...interfaces import KeyedData, KeyedDataAction, KeyedDataSchema, Scalar, Vector
+from ..keyedData.summaryStatistics import SummaryStatisticAction
+from .selectors import RangeSelector
+
+
+class CalcBinnedStatsAction(KeyedDataAction):
```

Comment:
Well, I guess I don't know how to politely tell everyone else that they're wrong because the dev guide says snake_case should be the default in primarily new code, which this package definitely is (and analysis_drp was too). None of it should have been written in camelCase in the first place.